Readme.md

```diff
@@ -25,34 +24,53 @@ See http://dev.maxmind.com/geoip/legacy/csv/ for a more exact rundown on some of
 
 There are two Meteor Methods provided that can be called from the client or server. Both return the same location info as returned by `UserLocation.get()`, documented above.
 
-```
+```js
 Meteor.call('UserLocation/get', callback) // IP lookup for the connected user
 Meteor.call('UserLocation/getForIp', ip, callback) // IP lookup for any ip
 ```
 
 Here is a short example:
 
-```
-Meteor.call('UserLocation/getForIp', '46.19.37.108', function(err,res){
-    console.log(err, res)
+```js
+Meteor.call('UserLocation/getForIp', '46.19.37.108', function (err, res) {
+    console.log(err, res);
 });
 ```
 
+## Paid maxmind service
 
-## HTTPS and Proxy Support
+To use the maxmind api to determine the user's location you have to provide your `userId` and `licenseKey` in your [Meteor settings](https://docs.meteor.com/api/core.html#Meteor-settings):
 
-If your server uses HTTPS or an internal proxy setup (via nginx etc.), the auto-detected IP will be inaccurate. In this case you will need to configure the HTTP_FORWARDED_COUNT enviroment variable on the server instance.
+```js
+{
+    // ...
+    "public": {
+        // ...
+    },
+    "mindmax": {
+        "userId": "YOUR_USER_ID",
+        "licenseKey": "YOUR_LICENSE_KEY"
+    }
+}
+```
 
-See the `clientAddress` section at http://docs.meteor.com/#/full/meteor_onconnection for more information.
+You can then call a Meteor method to retrieve the data:
+```js
+Meteor.call('UserLocation/getForIpWithService', ip, callback);
```

Comment:
witch service? What do you think about making this clear: `loopUpWithMaxmind`or something similar: `getIpWithMaxmind`